index.d.ts

```diff
@@ -157,28 +258,40 @@ declare const fileType: {
 
 	const url = 'https://assets-cdn.github.com/images/spinners/octocat-spinner-32.gif';
 
-	http.get(url, response => {
-		response.on('readable', () => {
+	http.get(url response => {
+		response.on('readable' () => {
```

Comment:
Are all these removed commas a bad find-and-replace?

Reply:
Yes, reverted it.